Geospatial.md

```diff
@@ -162,3 +192,18 @@ The axis order of the coordinates in WKB and bounding box stored in Parquet
 follows the de facto standard for axis order in WKB and is therefore always
 (x, y) where x is easting or longitude and y is northing or latitude. This
 ordering explicitly overrides the axis order as specified in the CRS.
+
+# Invalid geospatial values
+
+An invalid geospatial value refers to any of the following cases:
+
+* `null`: A null value in Parquet.
+* A non-`null` value that are encoded in a valid WKB or bounding box format 
+  but are not considered valid under this specification, including:
+  * `NaN`: Not a Number. For example, `POINT EMPTY` in WKB is represented by a 
+    `Point` with each ordinate value set to an IEEE-754 quiet NaN value.
+  * `Empty geometries`: Geometries explicitly marked as empty in WKB using 
```

Comment:
How to detect `Empty geometries` when collecting the bbox in Parquet? I still don't know what to do if I were to implement this rule.

Reply:
This is really a WKB reader-specific behavior. I added the following line:

> While different WKB readers may interpret such values differently, the resulting output should be treated as invalid